logging/lib/src/extension.dart

```diff
@@ -28,6 +28,7 @@ extension LogRecordX on LogRecord {
       level: level.toSentryLevel(),
       message: SentryMessage(message),
       throwable: error,
+      // ignore: deprecated_member_use
```

Comment:
Hello, I would like to see PR that migrates the current variables in extra so that I can replicate the same steps on our app that has a custom implementation of `sentry_logging`

is is ok to attach specific error data to the suggested `context`?

Reply:
yes, it's preferred actually https://docs.sentry.io/platforms/python/enriching-events/context/#additional-data
Although `extra` still works, and its probably not going to be removed any time soon, it's a soft deprecation as of now.